shared_model/cryptography/blob.hpp

```diff
@@ -39,12 +40,19 @@ namespace shared_model {
        * @param blob - string to create blob from
        */
       explicit Blob(const std::string &blob) : blob_(blob) {
-        std::stringstream ss;
-        ss << std::hex << std::setfill('0');
-        for (const auto &c : blob_) {
-          ss << std::setw(2) << (static_cast<int>(c) & 0xff);
-        }
-        hex_ = ss.str();
+        hex_ = iroha::bytestringToHexstring(blob_);
+      }
+
+      /**
+       * Creates new Blob object from provided hex string
+       * @param hex - string in hex format to create Blob from
+       * @return created Blob object
+       */
+      static Blob fromHexString(const std::string &hex) {
+        using iroha::operator|;
+        Blob b("");
```

Comment:
Need to write in the documentation about default case, when passed hex string is broken.